thoth/user_api/api_v1.py

```diff
@@ -394,6 +396,111 @@ def list_python_package_indexes():
     return graph.python_package_index_listing()
 
 
+def post_build(
+    build_detail: dict,
+    debug: bool = False,
+    registry_user: str = None,
+    registry_password: str = None,
+    environment_type: str = None,
+    origin: str = None,
+    registry_verify_tls: bool = True,
+    force: bool = False,
+):
+    """Run analysis on a build."""
+    response = {"base_image_analysis": {}, "output_image_analysis": {}, "build_log_analysis": {}}
```

Comment:
Using None to signalize no analysis was performed? So default response would become:

```python
    response = {"base_image_analysis": None, "output_image_analysis": None, "build_log_analysis": None}
```

Reply:
Not sure about this.

Reply:
Ideally even if the analysis of any one of these is not present, i think we should give `analysis_id:None`

Reply:
+1. I was targetting "invalid request" where a user does not provide any of the three - then its basically a noop operation as user-api does not perform any action.

Reply:
yes, for that I have now explicitly returned 400, bad request error.

Reply:
Ah, ok. Now I see adjusted code :+1: